rhel7/checks/oval/grub2_enable_fips_mode.xml

```diff
@@ -12,9 +12,12 @@
       <extend_definition comment="prelink disabled" definition_ref="disable_prelink" />
       <extend_definition comment="package dracut-fips installed" definition_ref="package_dracut-fips_installed" />
       <criteria operator="OR">
-        <criterion test_ref="test_grub2_enable_fips_mode" comment="check for fips=1 in /etc/default/grub via GRUB_CMDLINE_LINUX" />
         <criteria operator="AND">
-          <criterion test_ref="test_grub2_default_exists" comment="check for GRUB_CMDLINE_LINUX_DEFAULT exists in /etc/default/grub" />
+          <extend_definition definition_ref="test_grub2_default_exists" comment="check for GRUB_CMDLINE_LINUX_DEFAULT exists in /etc/default/grub" />
+          <criterion test_ref="test_grub2_enable_fips_mode" comment="check for fips=1 in /etc/default/grub via GRUB_CMDLINE_LINUX" />
+        </criteria>
```

Comment:
@cipherboy please remove the changes in lines 16-18.

Reply:
why?

Reply:
@redhatrises Sorry, misread that test. I thought it was `/etc/default/grub` exists. My bad!